internal/pkg/rbldap/search.go

```diff
@@ -0,0 +1,74 @@
+package rbldap
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+
+	"github.com/urfave/cli"
+)
+
+// Search command for cli app
+func Search(ctx *cli.Context) error {
+	mail := filter("altmail", ctx.String("mail"))
+	id := filter("id", ctx.String("id"))
+	var re = regexp.MustCompile(`\ `)
```

Comment:
Any particular reason why this uses `var` instead of `:=`? I have nothing against one method or another unless they are consistent or there is a reason.